.gitignore

```diff
@@ -9,3 +9,7 @@
 
 # nyc
 /.nyc_output
+
+.DS_Store
+
+.env
```

Comment:
might not need this, otherwise lgtm

Reply:
Definitely need to ignore .DS_Store, .env is for ✨someday✨